tcl/org.testeditor.tcl.dsl.tests/src/org/testeditor/tcl/dsl/jvmmodel/SimpleTclGeneratorIntegrationTest.xtend

```diff
@@ -66,13 +66,16 @@ class SimpleTclGeneratorIntegrationTest extends AbstractTclGeneratorIntegrationT
 				- assert baz = "fix"
 				- assert book
 				- assert mak."key with spaces" = "fox"
-				- assert ! foo
-				- assert baz <> "fix"
+				- assert false foo
```

Comment:
Not really. The two lines 'assert ! foo' and 'assert false foo' are equivalent ('false' just an alias for '!' in this case).